pkg/ciliumenvoyconfig/cell.go

```diff
@@ -87,6 +92,13 @@ func registerCECK8sReconciler(params reconcilerParams) {
 		},
 	})
 
+	reconciler.registerResourceWithSyncFn(context.Background(), k8sAPIGroupCiliumEnvoyConfigV2, func() bool {
+		return reconciler.cecSynced.Load()
+	})
+	reconciler.registerResourceWithSyncFn(context.Background(), k8sAPIGroupCiliumClusterwideEnvoyConfigV2, func() bool {
```

Comment:
It's not ideal that we're using `context.Background()` here. I think this might leak go routines if this cell is ever included by the Cell-based tests in `test/controlplane`

Can we use a context that is cancelled once the cell is stopped (like the CiliumNetwork policy watcher does)?

Reply:
Done :-)